gwcli/gateway.py

```diff
@@ -312,6 +312,12 @@ def clear_config(self, gateway_group):
         # last to ensure we don't fall foul of the api auth check
         gw_list = [gw_name for gw_name in gateway_group
                    if isinstance(gateway_group[gw_name], dict)]
+
+        if this_host() not in gw_list:
+            self.logger.error("Executor({}) is not in gateway list: "
+                              "{}".format(this_host(), gw_list))
```

Comment:
Is this an error? It seems like it can happen under normal conditions if the user has not fully set things up but just wants to clear the config.

Maybe just a logger.warning or logger.info?

Reply:
maybe logger.warning is better, I will update it later